cmd/agent/install_mac_os.sh

```diff
@@ -455,6 +468,15 @@ else
     $sudo_cmd launchctl kickstart "system/$service_name"
 fi
 
+# Set up and start the system-probe service if this version includes support for it
+if [ -f "${etc_dir}/com.datadoghq.sysprobe.plist.example" ]; then
+    printf "\033[34m\n* Setting up system-probe ($sysprobe_service_name) as a systemwide LaunchDaemon ...\n\n\033[0m"
+    $sudo_cmd mv "${etc_dir}/com.datadoghq.sysprobe.plist.example" "${sysprobe_servicefile_name}"
+    $sudo_cmd chown "0:0" "$sysprobe_servicefile_name"
+    $sudo_cmd launchctl load -w "$sysprobe_servicefile_name"
```

Comment:
We have a mix of `${sysprobe_servicefile_name}` and `$sysprobe_servicefile_name` there, should we try to be consistent?